ports/matio/portfile.cmake

```
@@ -27,10 +27,23 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 
+set(prefix "${CURRENT_INSTALLED_DIR}")
+set(exec_prefix \$\{prefix\})
```

Such patterns are simpler with 
```suggestion
set(exec_prefix [[${prefix}]])
```